src/Sylius/Component/Order/Repository/OrderRepositoryInterface.php

```diff
@@ -40,7 +40,7 @@ public function findCartById($id): ?OrderInterface;
     /**
      * @return array|OrderInterface[]
      */
-    public function findCartsNotModifiedSince(\DateTimeInterface $terminalDate): array;
+    public function findCartsNotModifiedSince(\DateTimeInterface $terminalDate, int $limit = -1): array;
```

Comment:
I know that this is not covered by the BC Promise, but I would rather add a new method here and deprecate this one.
This method might be already implemented/overridden in end projects.

Reply:
We can consider doing it like the Symfony does, comment this additional parameter like here https://github.com/symfony/serializer/blob/6.1/Normalizer/NormalizerInterface.php#L50 and add it in the phpdoc.